### PR TITLE
Code and Mapping

### PR DESCRIPTION
**Pull request by Noightmare**
This is a long list of stuff that I spent like four or so hours doing, doing a quick summary.
Code:
Moved around some of the magazines on other disks to be more in order/proper.
Added a few more mags and ammo boxes to the Blackshield sovereign disk to make it more 'Sovereign', they will still need to go to Guild or Marshals for more ammo/different ammo.

Mapping:
Added a CURSED bathroom to the Blackshield Barracks.
Made the other way to tunnel into the colony just as dangerous as the other tunnel by the crematorium.
Littered the colony grounds between perimeter and colony with more trapdoor burrows
Moved organ freezer from Shield to Corpsman triage, removed box of latex gloves, replaced all five first aid kits with a singular Blackshield siege kit.
Slapped in a box of handcuffs into command armory for steward.
Gave BS their box of crayons.
Created a single way for someone to mine into BS on the staircase zone.
Uhhhhhhh forgetting whatever else I did. Have fun.

image

image

image

image

image

image

image

### CODE REVIEW

**Comment by Kazkin** [comment]
Code looks fine, but can you screenshot the map changes?

File: code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm [hunk 1]
```diff
@@ -152,19 +152,26 @@
 		/datum/design/autolathe/ammo/magnum_rubber,
 		/datum/design/autolathe/ammo/magnum_practice = 0,
 		/datum/design/autolathe/ammo/magnum_lethal,
+		/datum/design/autolathe/ammo/slkurtz,
+		/datum/design/autolathe/ammo/slkurtz_lethal,
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:156]: I dont think this is a good idea + this needs to be mirrored to marshals disk to be 1:1

Comment (Noightmare) [reply code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:156]: image
image
image
image
image
image
image
Hope the pictures work, if not, I'll just DM them to Kazkin.

As for the disks, eh, if we're gonna make them 1:1, I'd really RATHER not figure out on what gun to give to the Shield disk that would take a exceptionally large amount of points like the Nordwind does on the Marshal disk. Futhermore, Marshals do get a abundant more amount of disks than Shields do, but can make them 1:1 if you really want to Trilby.

Comment (r4d6) [reply code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:156]: The pictures work, but you posted them in the wrong place.

**Pull request merged**